src/components/AdminProfile/AdminProfile.jsx

```diff
@@ -93,65 +94,120 @@ const AdminProfile = () => {
-            placeHolder={lastName}
-            value={lastName}
-            setValue={setLastName}
-            editable
-            editState={editLast}
-            makeEditable={() => setEditLast(!editLast)}
-            handleCheckMarkClicked={handleCheckMarkClicked}
-          />
-          <TextInput inputLabel="Email" placeHolder={email} editState={false} />
-        </div>
-        <div className={styles['bottom-buttons']}>
-          <Button onClick={onOpenDeleteModal} colorScheme="red" size="lg" rightIcon={<DiReact />}>
-            Delete Account
-          </Button>
-          <Button
-            onClick={onOpenLogoutModal}
-            colorScheme="teal"
-            bg="#345E80"
-            fontSize="20px"
-            size="lg"
-            rightIcon={<DiReact />}
-          >
-            Logout
-          </Button>
-          <DeleteAccountModal isOpen={isOpenDeleteModal} onClose={onCloseDeleteModal} />
-          <LogoutModal isOpen={isOpenLogoutModal} onClose={onCloseLogoutModal} />
+          <div className={styles['profile-form']}>
+            {!isMobile && (
+              <div className={styles['logo-wrapper']}>
+                <img
+                  className={styles['fya-logo']}
+                  src={FYALogoLarge}
+                  alt="Find Your Anchor Logo"
+                />
+              </div>
+            )}
+
+            <TextInput
+              inputLabel="First Name"
+              placeHolder={firstName}
+              value={firstName}
+              setValue={setFirstName}
+              editable
+              editState={editFirst}
+              makeEditable={() => setEditFirst(!editFirst)}
+              handleCheckMarkClicked={handleCheckMarkClicked}
+            />
+            <TextInput
+              inputLabel="Last Name"
+              placeHolder={lastName}
+              value={lastName}
+              setValue={setLastName}
+              editable
+              editState={editLast}
+              makeEditable={() => setEditLast(!editLast)}
+              handleCheckMarkClicked={handleCheckMarkClicked}
+            />
+            <TextInput inputLabel="Email" placeHolder={email} editState={false} />
+            {isMobile && (
+              <div className={styles['bottom-buttons']}>
+                <Button
+                  onClick={onOpenLinkModal}
+                  color="white"
+                  bg="#1F2F38"
+                  fontSize="18px"
+                  size="lg"
+                  width={isMobile ? '100%' : 'auto'}
+                >
+                  Send Registration Link
+                </Button>
+                <Button
+                  onClick={onOpenDeleteModal}
+                  colorScheme="red"
+                  size="lg"
+                  width={isMobile ? '100%' : 'auto'}
+                >
+                  Delete Account
+                </Button>
+                <DeleteAccountModal isOpen={isOpenDeleteModal} onClose={onCloseDeleteModal} />
```

Comment:
The modal is still a little off due to the gray area behind it. It's filled on some mobile screens, but it doesn't fill the entire screen for other phone screens
<img width="352" alt="Screen Shot 2022-05-05 at 11 30 23 PM" src="https://user-images.githubusercontent.com/56140648/167079294-f8e4f4ee-a879-4d9c-b3e4-6a13ccfb7270.png">
.

Reply:
Marking this as resolved as it will be fixed in a future PR